frigate/ptz/onvif.py

```diff
@@ -486,7 +488,10 @@ def get_camera_status(self, camera_name: str) -> None:
 
         onvif: ONVIFCamera = self.cams[camera_name]["onvif"]
         status_request = self.cams[camera_name]["status_request"]
-        status = onvif.get_service("ptz").GetStatus(status_request)
+        try:
+            status = onvif.get_service("ptz").GetStatus(status_request)
+        except Exception as e:
```

Comment:
```suggestion
        except Exception:
```

Reply:
Aww come on, why did that test not fail before....

Reply:
because it didn't get that far, black was failing